parachain/runtime/snowbase/Cargo.toml

```diff
@@ -72,8 +71,8 @@ runtime-primitives = { path = "../../primitives/runtime", default-features = fal
 
 snowbridge-xcm-support = { path = "../../primitives/xcm-support", default-features = false }
 snowbridge-asset-registry = { path = "../../pallets/asset-registry", default-features = false }
-basic-channel = { path = "../../pallets/basic-channel", package = "snowbridge-basic-channel", default-features = false }
-incentivized-channel = { path = "../../pallets/incentivized-channel", package = "snowbridge-incentivized-channel", default-features = false }
+snowbridge-basic-channel = { path = "../../pallets/basic-channel", default-features = false }
```

Comment:
Should we prefix the other pallets with `snowbridge` too? Checking what we want the convention to be.

Reply:
I'd like to, but it's not a strong preference. It helps distinguish our pallets from others at a glance. I've made the change to the incentivized channel pallet already, but there are several others that don't have the prefix and only a few that do.

@vgeddes @alistair-singh Do you think it's worth having the `snowbridge-` prefix on our pallets? Is there an existing convention for when we use it?

Reply:
I'd say we should keep the snowbridge prefix. I've been bitten before, when a local crate in the monorepo clashed with a public crate on https://crates.io/, causing compilation failures.